docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,9 +1,18 @@
 = Hazelcast Platform Operator
 :!page-pagination:
-:description: Hazelcast Platform Operator automates common management tasks such as configuring, creating, scaling, and recovering Hazelcast clusters on Kubernetes and Red Hat OpenShift. By taking care of manual deployment and life-cycle management, Hazelcast Platform Operator makes it simpler to work with Hazelcast clusters.
+:description: By eliminating the need for manual deployment and life-cycle management, Hazelcast Platform Operator simplifies working with Hazelcast clusters on Kubernetes and Red Hat OpenShift.
 
 {description}
 
+Hazelcast Platform Operator allows your development and DevOps teams to automate common management tasks for your Hazelcast clusters on Kubernetes and Red Hat OpenShift. For example, you can use Hazelcast Platform Operator to automate the following tasks:
+
```

Comment:
@rebekah-lawrence , the last couple of words of the description text and the sentence after it are the same. The flow or reading made me feel a duplication since they are very close to each other.

Reply:
Yup, here was my logic: 

- The first sentence was already too long (24 words), so I didn't want to add something like ", such as the following"
- They are a duplication as one is introducing examples of the other; it is difficult to avoid the duplication and retain the sense of what is being introduced
- People scan, so the duplication also helps to ensure they do digest what is being said. Also, I'd rather use the same words than paraphrase for variety so that people don't have to think while they scan

Any suggestions to improve it?